configure.ac

```diff
@@ -923,6 +923,20 @@ AS_IF([test "x$enable_slingshot" = xyes],[
         [AC_MSG_ERROR([libcxi or its headers not found])])
 ])
 
+
+AC_ARG_ENABLE([libzfs],
+    [AS_HELP_STRING([--enable-libzfs], [require the libzfs related plugins @<:@default=check@:>@])],
+    [],
+    [enable_libzfs="check"])
+AM_CONDITIONAL([ENABLE_LIBZFS], [test "x$enable_libzfs" != xno])
```

Comment:
This needs to be the last thing in this hunk of added lines. The test needs to check that both enable_libzfs != xno and also have_libzfs = xyes.